rubble/src/att/server.rs

```diff
@@ -197,10 +203,17 @@ impl<A: AttributeProvider> AttributeServer<A> {
                         self.attrs.for_attrs_in_range(
                             HandleRange::new(*handle, *handle),
                             |_provider, attr| {
-                                let value = if writer.space_left() < attr.value.as_slice().len() {
-                                    &attr.value.as_slice()[..writer.space_left()]
+                                // FIXME return if attribute is not readable
```

Comment:
This callback can't return the proper error type with the current API design due (see comment).